micrometer-core/src/main/java/io/micrometer/core/instrument/placeholder/RemovePlaceholdersNamingConvention.java

```diff
@@ -0,0 +1,38 @@
+package io.micrometer.core.instrument.placeholder;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.instrument.util.StringUtils;
+import io.micrometer.core.lang.Nullable;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * A naming convention that removes all placeholders from the metric name. Configured in
+ * non-hierarchical registries when using the placeholders mechanism. Metrics
+ * registered in non-hierarchical registries don't have tags encoded in their name, because
+ * they support tags explicitly.
+ *
+ * @author Piotr Betkier
+ */
+class RemovePlaceholdersNamingConvention implements NamingConvention {
+
+    private static final Pattern PLACEHOLDER = Pattern.compile("\\{.*\\}");
```

Comment:
`*` is greedy by default in java regular expressions. So in the case of `fixed.{var}.fixed2.{var2}.fixed3` will be processed to `fixed.fixed3`.